com.zsmartsystems.zigbee.autocode/src/main/java/com/zsmartsystems/zigbee/autocode/ZigBeeZclClusterGenerator.java

```diff
@@ -397,6 +397,7 @@ private void generateZclClusterClasses(ZigBeeXmlCluster cluster, String packageR
 
             out.println("     * @return the {@link Future<CommandResult>} command result future");
             out.println("     */");
+            out.println("    @Deprecated");
```

Comment:
Should we have a statement in the Javadoc like following?
`@Deprecated use generic sendCommand/sendResponse methods in ZclCluster instead`